Source/Extensions/Blazorise.Icons.FontAwesome/FontAwesomeIcons.cs

```diff
@@ -0,0 +1,1011 @@
+        public static readonly string Dharmachakra = "dharmachakra";
+        public static readonly string Diagnoses = "diagnoses";
+        public static readonly string Dice = "dice";
+        public static readonly string DiceD20 = "dice-d20";
+        public static readonly string DiceD6 = "dice-d6";
+        public static readonly string DiceFive = "dice-five";
+        public static readonly string DiceFour = "dice-four";
+        public static readonly string DiceOne = "dice-one";
+        public static readonly string DiceSix = "dice-six";
+        public static readonly string DiceThree = "dice-three";
+        public static readonly string DiceTwo = "dice-two";
+        public static readonly string DigitalTachograph = "digital-tachograph";
+        public static readonly string Directions = "directions";
+        public static readonly string Disease = "disease";
+        public static readonly string Divide = "divide";
+        public static readonly string Dizzy = "dizzy";
+        public static readonly string Dna = "dna";
+        public static readonly string Dog = "dog";
+        public static readonly string DollarSign = "dollar-sign";
+        public static readonly string Dolly = "dolly";
+        public static readonly string DollyFlatbed = "dolly-flatbed";
+        public static readonly string Donate = "donate";
+        public static readonly string DoorClosed = "door-closed";
+        public static readonly string DoorOpen = "door-open";
+        public static readonly string DotCircle = "dot-circle";
+        public static readonly string Dove = "dove";
+        public static readonly string Download = "download";
+        public static readonly string DraftingCompass = "drafting-compass";
+        public static readonly string Dragon = "dragon";
+        public static readonly string DrawPolygon = "draw-polygon";
+        public static readonly string Drum = "drum";
+        public static readonly string DrumSteelpan = "drum-steelpan";
+        public static readonly string DrumstickBite = "drumstick-bite";
+        public static readonly string Dumbbell = "dumbbell";
+        public static readonly string Dumpster = "dumpster";
+        public static readonly string DumpsterFire = "dumpster-fire";
+        public static readonly string Dungeon = "dungeon";
+        public static readonly string Edit = "edit";
+        public static readonly string Egg = "egg";
+        public static readonly string Eject = "eject";
+        public static readonly string EllipsisH = "ellipsis-h";
+        public static readonly string EllipsisV = "ellipsis-v";
+        public static readonly string Envelope = "envelope";
+        public static readonly string EnvelopeOpen = "envelope-open";
+        public static readonly string EnvelopeOpenText = "envelope-open-text";
+        public static readonly string EnvelopeSquare = "envelope-square";
+        public static readonly string Equals = "equals";
```

Comment:
this should have the '''new''' keyword

Reply:
I didn't notice that. Thank you!

Reply:
In my generator, I used reflection so that if any member of ```object``` needs to be redefined, it includes the ```new``` keyword

Reply:
I have fixed the generator so the `new `keyword should now work.